package/kedro_viz/server.py

```diff
@@ -26,7 +26,7 @@ def populate_data(
     if creating an api app from project.
     """
 
-    data_access_manager.add_catalog(catalog, pipelines)
+    data_access_manager.add_catalog(catalog)
```

Comment:
why don't we need to pass the `pipelines` anymore?  @rashidakanchwala

Reply:
Yup! Initially, we passed only the catalog. However, we encountered issues with dataset factories not working with Experiment Tracking, so we had to pass pipelines to resolve the dataset factory issue. Now that Experiment Tracking is no longer in use and DataCatalog 2.0 handles this, we can clean up this part and revert to passing just the catalog.